evap/student/templates/student_index.html

```diff
@@ -87,7 +87,9 @@
                                             <a href="{% url "student:vote" course.id %}" class="btn btn-sm btn-primary">{% trans "Evaluate" %}</a>
                                         {% endif %}
                                     {% endif %}
-                                    {% if course.grade_documents.count > 0 and can_download_grades %}
+                                    {% if course.grade_documents.count = 1 and can_download_grades %}
```

Comment:
did you talk to @janno42 about this and the other similar comment or address this somewhere else?